src/coredata.cs

```diff
@@ -1686,6 +1702,23 @@ interface NSCoreDataCoreSpotlightDelegate
 
 		[Export ("searchableIndex:reindexSearchableItemsWithIdentifiers:acknowledgementHandler:")]
 		void ReindexSearchableItems (CSSearchableIndex searchableIndex, string[] identifiers, Action acknowledgementHandler);
+
+		[Async]
+		[NoWatch, NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("deleteSpotlightIndexWithCompletionHandler:")]
+		void DeleteSpotlightIndex (Action<NSError> completionHandler);
+
+		[NoWatch, NoTV, Mac (12, 0), iOS (15, 0), MacCatalyst (15,0)]
+		[Export ("indexingEnabled")]
+		bool IndexingEnabled { [Bind ("isIndexingEnabled")] get; }
+
+		[NoWatch, NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("startSpotlightIndexing")]
+		void StartSpotlightIndexing ();
+
+		[NoWatch, NoTV, Mac (12,0), iOS (15,0)]
+		[Export ("stopSpotlightIndexing"), MacCatalyst (15,0)]
```

Comment:
it's legit - but it's confusing to mix availability and `[Export]` inside the same `[...]`